src/parser/xasl_generation.c

```diff
@@ -11672,7 +11672,7 @@ pt_fix_first_term_func_index_for_iss (PARSER_CONTEXT * parser, QO_INDEX_ENTRY *
   seg = QO_ENV_SEG (index_entryp->terms.env, index_entryp->seg_idxs[1]);
   head = QO_SEG_HEAD (seg);
   spec = head->entity_spec;
-  class_name = (char *) spec->info.spec.range_var->info.name.original;
+  class_name = parser_print_tree (parser, spec->info.spec.entity_name);
```

Comment:
I found class_name is got like the following in many other codes. Is it exactly the same? 
```
classname = node->info.spec.entity_name->info.name.original;
```

Reply:
Okay. Using parser_print_tree(), it seems that even unnecessary things will be printed. I'll fix it.

Reply:
Isn't spec->info.spec.range_var different from node->info.spec.entity_name?
If an alias is used in the query, it seems appropriate to use spec->info.spec.range_var.

Reply:
This is a bug where a table name is required, but an alias is used.